packages/react-components/react-jsx-runtime/etc/react-jsx-runtime.api.md

```diff
@@ -8,7 +8,7 @@ import { Fragment } from 'react';
 import * as React_2 from 'react';
 
 // @public (undocumented)
-export function createElement<P extends {}>(type: React_2.ElementType<P>, props?: P | null, ...children: React_2.ReactNode[]): React_2.ReactElement<P> | null;
+export function createElement<P extends {}>(type: React_2.ElementType<P>, props?: P | null, ...children: React_2.ReactNode[]): React_2.ReactElement<P>;
```

Comment:
Curiosity: why did `null` dissappear?

Reply:
To follow up on  `@type/react` `createElement`  signature, it doesn't return null never